src/bornhack/oauth_validators.py

```diff
@@ -50,8 +50,8 @@ def get_additional_claims(self, request) -> dict[str, str | list[dict[str, str]]
                 # standard OIDC claims
                 "email": request.user.email,
                 "email_verified": True,
-                "nickname": request.user.profile.get_public_credit_name,
                 "updated_at": int(request.user.profile.updated.timestamp()),
+                "username": request.user.username,
```

Comment:
nickname is a standard claim name, username is not. See https://openid.net/specs/openid-connect-core-1_0.html#Claims for the list of standard claims.

> Casual name of the End-User that may or may not be the same as the given_name. For instance, a nickname value of Mike might be returned alongside a given_name value of Michael.

Reply:
We set both `nickname` and `preferred_username` below, maybe we just want to delete this line?

Reply:
Done